components/ui/AppCta.vue

```diff
@@ -37,5 +37,26 @@ export default class extends Vue {
   & :first-child {
     margin-right: $layout-02;
   }
+  svg {
+      overflow: visible;
+  }
+  &[target=_blank] {
+    path:nth-child(2) {
```

Comment:
Provide a meaningful SASS name to this selector, inside the corresponding modifier, such as:

```
&_is-external {
  $arrow-path: "path:nth-child(2)";
}
```